docs/en/observability/uptime-set-up-choose-project-monitors.asciidoc

```diff
@@ -89,7 +177,47 @@ with the {kib} URL for the deployment from which to fetch available locations.
 ====
 
 [discrete]
-[[uptime-app-view-in-kibana-project]]
 == View in {kib}
 
-include::uptime-set-up-app.asciidoc[tag=next]
\ No newline at end of file
+Then, go to the {uptime-app} in {kib}. You should see your newly pushed monitors running. You can also go to the *Monitor Management* page to see the monitors' configuration settings.
+
+[discrete]
+== Recommended workflow
+
+After your project is set up, there are some best practices you can implement to manage project monitors effectively.
+
+First, it's recommended that you version control all files in Git. If your project is not already in a version controlled directory add it now, and push it to your Git host.
+
+While it can be convenient to run the `push` command directly from your workstation,
+especially when setting up a new project, it is not recommended for production environments.
+
+Instead, we recommended that you:
+
+1. Develop and test changes locally
+2. Create a pull request for all config changes
+3. Have your CI service automatically verify the PR by running `npx @elastic/synthetics .`
+4. Have a human approve the pull request
+5. Merge the pull request
+6. Have your CI service automatically deploy the change by running `npx @elastic/synthetics push` after changes are merged
+
+The exact implementation details will depend on the CI system and Git host you use. 
+You can reference the sample GitHub configuration file that is included in the `.github`
+directory when you create a new project.
+
+// [discrete]
+// [[synthetics-ci]]
+// === Run on CI
+
+// You can also run a synthetic project on your CI environment. Elastic's synthetics runner can output results in a few different formats, including JSON and JUnit
+// (the standard format supported by most CI platforms).
+
+// If any of your journeys fail, the synthetics agent it will yield a non-zero exit code, which most CI systems pick up as a failure by default.
+
+// You can see an example using GitHub Actions in the
+// https://github.com/elastic/synthetics-demo/blob/main/.github/workflows/run-synthetics.yml[elastic/synthetics-demo] repository.
+// This example sets up a job that executes the synthetics runner and tells the runner to yield results in a JUnit format.
```

Comment:
@andrewvc I dropped in the content from the [Run on CI](https://www.elastic.co/guide/en/observability/current/synthetic-run-tests.html#synthetics-ci) section here because I thought it could be integrated into the recommended workflow. Is there any detail in the commented out text that you would integrate into the text above?